# -------------------------------------------
#   Waypoint Planner -- parameters
# -------------------------------------------

search_radius               : 75
search_fov                  : 360
local_pose_topic            : /mavros/global_position/local
utm_location_odom_topic     : /lhex2/advanced_navigation/aidtr_sensor/odom
local_odom_topic            : /mavros/global_position/local
autonomy_server_topic       : /nrec/zone_recon_autonomy
search_polygon_topic        : /search_polygon/specification
search_polygon_viz_topic    : search_polygon/visualisation
